Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManager.cpp

```diff
@@ -573,6 +581,21 @@ namespace AzToolsFramework
         return AZ::Success();
     }
 
+    MenuManagerOperationResult MenuManager::QueueRefreshForMenusContainingSubMenu(const AZStd::string& subMenuIdentifier)
+    {
+        auto menuIterator = m_subMenusToMenusMap.find(subMenuIdentifier);
+
+        if (menuIterator != m_subMenusToMenusMap.end())
```

Comment:
suggestion: It might be nice to switch this to an if initializer to minimize the scope of `menuIterator`

Reply:
Amazing thanks! 🙂